extensions/stats/plugin.cc

```diff
@@ -450,9 +450,9 @@ bool PluginRootContext::initializeDimensions(const json& j) {
   Metric build(MetricType::Gauge, absl::StrCat(stat_prefix, "build"),
                {MetricTag{"component", MetricTag::TagType::String},
                 MetricTag{"tag", MetricTag::TagType::String}});
+  std::string istio_version = flatbuffers::GetString(local_node.istio_version()) == "" ? absl::StrCat(unknown, ";") : absl::StrCat(istio_version, ";");
```

Comment:
This seems wrong.. istio_version is used before it is defined.

Reply:
@bianpengyuan Thanks Peter~ I think it`s ok now